can/interfaces/pcan/basic.py

```diff
@@ -658,13 +658,15 @@ def GetValue(
           A touple with 2 values
         """
         try:
-            if Parameter == PCAN_API_VERSION or Parameter == PCAN_HARDWARE_NAME or Parameter == PCAN_CHANNEL_VERSION or Parameter == PCAN_LOG_LOCATION or Parameter == PCAN_TRACE_LOCATION or Parameter == PCAN_BITRATE_INFO_FD or Parameter == PCAN_IP_ADDRESS:
+            if Parameter in (PCAN_API_VERSION, PCAN_HARDWARE_NAME, PCAN_CHANNEL_VERSION,
```

Comment:
nit: use a set with `{PCAN_API_VERSION, ...}` instead of a tuple

Reply:
Sounds good. I didn't know there was a special syntax for it. 👀